core/src/main/scala/com/salesforce/op/evaluators/OpMultiClassificationEvaluator.scala

```diff
@@ -94,6 +98,25 @@ private[op] class OpMultiClassificationEvaluator
 
   def setThresholds(v: Array[Double]): this.type = set(thresholds, v)
 
+  def setTopKCm(v: Int): this.type = set(topKCm, v)
```

Comment:
I think we should call these something different since we already have confusion between `topK` and `topN` in the multiclass metrics. How about a more self-documenting name like `confMatrixNumClasses` for `topKCm` and `confMatrixMinSupport` for `topNCm`?